ext/RMagick/extconf.rb

```diff
@@ -3,7 +3,12 @@
 
 module RMagick
   class Extconf
-    require 'rmagick/version'
+    if RUBY_VERSION < '1.9.2'
+      require File.expand_path('../../lib/rmagick/version',
+                               File.dirname(__FILE__))
+    else
+      require_relative '../../lib/rmagick/version'
+    end
```

Comment:
If these both work, then you only need the 1.9.2 form.  

This is a load path issue, though.  If you can duplicate the failure, we'll need to ensure we have `$LOAD_PATH.unshift(lib_dir) unless $LOAD_PATH.include?(lib_dir)` before the extconf.rb is evaluated.

The reason the load path is used is due to the way the gem builds.